staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/initializer.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+import (
+	"context"
+
+	"k8s.io/apiserver/pkg/admission"
+)
+
+type CELPolicyEvaluator interface {
+	Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error
+	HasSynced() bool
+}
+
+// Creates a plugin initializer which dependency injects a singleton
+// cel admission controller into the plugins which desire it
+func NewPluginInitializer(validator CELPolicyEvaluator) *PluginInitializer {
+	return &PluginInitializer{validator: validator}
+}
+
+// WantsCelAdmissionController gives the ability to have the shared
+// CEL Admission Controller dependency injected at initialization-time.
+type WantsCELPolicyEvaluator interface {
+	SetCELPolicyEvaluator(CELPolicyEvaluator)
+	admission.InitializationValidator
```

Comment:
Why embed `InitializationValidator` inside `WantsCELPolicyEvaluator`? Would it be better to define another interface for both `WantsCELPolicyEvaluator` and `InitializationValidator`.

Reply:
You are right. I think this is included by a mistake.